syz-manager/covfilter_test.go

```diff
@@ -50,4 +50,7 @@ func TestCreateBitmap(t *testing.T) {
 		0x81000102: 1,
 	}
 	createCoverageBitmap(target, pcs)
+	// Test nil bitmap.
+	pcs = nil
+	createCoverageBitmap(target, pcs)
```

Comment:
Does this fail w/o the fix? I guess we needs to check something here, or it won't prevent similar bugs in future.

Reply:
W/o the fix this created a bitmap with a full header but empty size, causing an error when the executor checked the bitmap flag against the size of the bitmap (https://github.com/google/syzkaller/blob/master/executor/cov_filter.h#L43).

Reply:
I mean: if this bug is re-introduiced tomorrow, will this test fail?

Reply:
Good point, thanks! I added a fix for this test here:
https://github.com/google/syzkaller/pull/4029

It looks like there's one test which would fail (https://github.com/google/syzkaller/pull/4029/files#diff-8f9f23150115998291b4bf7d946b35f91694b3bd2ce37ac3e4d6ed076148aee3L34), but the rest do not have any error checking. I can add this as well.